Mathlib/Topology/Bornology/BoundedOperation.lean

```diff
@@ -107,4 +173,32 @@ noncomputable instance : BoundedSub ℝ≥0 where
     simp only [NNReal.ball_zero_eq_Ico, ← z_eq, Set.mem_Ico, zero_le, true_and, gt_iff_lt] at *
     exact tsub_lt_of_lt key
 
+instance : BoundedMul ℝ≥0 where
+  isBounded_mul := by
+    intro s t hs ht
```

Comment:
The proof looks a little bit too complicated to me. Can you deduce it from properness and continuity of multiplicationN

Reply:
Thanks again for an obvious improvement! I did that.

I did not find an existing `instance ProperSpace ℝ≥0`, however. And when I created one, `#find_home` suggested `Mathlib.Analysis.Normed.Field.Basic`, which does not feel like the right place for the NNReal-specific lemma... The most natural place to look for it would be `Mathlib.Topology.Instance.NNReal`, but I don't think it has sufficient imports. For now, this instance is in the new file `Mathlib.Topology.Bornology.BoundedOperation`, although it feels like a wrong place, too

Reply:
I'd put it in `Mathlib.Topology.Instances.NNReal`. There is a missing import for that, `import Mathlib.Topology.MetricSpace.Isometry`, but that's definitely a reasonable import.